src/data/faq_de.json

```
@@ -3679,7 +3706,7 @@
                 {
                     "term": "TAN",
                     "anchor": "tan",
-                    "description": "10-stellige Buchstaben-Zahlenkombination zur Meldung positiver Testergebnisse. Diese sind über die TAN-Hotline anzufragen. <b>National:</b> 0800 7540002 und <b>international:</b> +49 30 498 75402"
+                    "description": "<p>Aktualisiert am ?????r.</p> 10-stellige Buchstaben-Zahlenkombination zur Meldung positiver Testergebnisse. Diese sind über die TAN-Hotline anzufragen. <b>National:</b> 0800 7540002 und <b>international:</b> +49 30 498 75402. <p>Beachten Sie bitte, dass ab der Version 3.0 der Corona-Warn-App keine teleTAN mehr benötigt wird, um ein positives Testergebnis zu teilen, welches nicht in die Corona-Warn-App übermittelt wurde. Daher wird der Betrieb der TAN-Hotline zum ?????h eingestellt.</p><a href='#warn_without_tan' target='_blank'>Wie warne ich ohne teleTAN?</a>"

```

Specified placeholder was ??????h, one ? is missing
Update note should be in bold text and without a "." at the end of it, just like in FAQ entries
```suggestion
                    "description": "<p><b>Aktualisiert am ?????r</b></p> 10-stellige Buchstaben-Zahlenkombination zur Meldung positiver Testergebnisse. Diese sind über die TAN-Hotline anzufragen. <b>National:</b> 0800 7540002 und <b>international:</b> +49 30 498 75402. <p>Beachten Sie bitte, dass ab der Version 3.0 der Corona-Warn-App keine teleTAN mehr benötigt wird, um ein positives Testergebnis zu teilen, welches nicht in die Corona-Warn-App übermittelt wurde. Daher wird der Betrieb der TAN-Hotline zum ??????h eingestellt.</p><a href='#warn_without_tan' target='_blank'>Wie warne ich ohne teleTAN?</a>"
```